docs/_data/prague-2021-schedule.yaml

```diff
@@ -0,0 +1,79 @@
+writing_day:
+- time: '10:00'
+  title: 'Platform opens'
+- time: '11:00'
+  title: Morning introduction - Introduction to Sprints & Project Introductions
+- time: '11:15'
+  title: Team up and start working
+- time: '14:30'
+  title: Lunch Break
+- time: '19:00'
+  title: End of Writing Day
+
+talks_day1:
+- time: '9:00'
+  title: Platform opens
+- time: '10:00'
+  title: Write the Docs Team - Introduction to Write the Docs
+- time: '10:45'
+  title: <b>Unconference Starts</b>
+- time: '10:45'
+  slug: hitchhiker-s-guide-to-documentation-tools-and-processes-lukas-reu-ner
+- time: '11:25'
```

Comment:
Is this time slot shorter intentionally? Feels like it would hurt the Q&A time, unless you know that the talk itself is short....